packages/components/src/FormatFile/FormatFile.css

```diff
@@ -51,12 +51,61 @@
   margin: 0 auto;
 }
 
-.progress {
+.fileContentWrapper {
+  display: flex;
+  width: inherit;
+  height: inherit;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.thumbailIconWrapper {
+  align-self: center;
+}
+
+.thumbnailFilename {
   display: flex;
   position: absolute;
-  top: 0;
+  bottom: var(--space-small);
+  left: var(--space-small);
+}
+
+.thumbnailFilenameSmall {
+  display: flex;
+  position: absolute;
+  bottom: var(--space-smaller);
+  left: var(--space-smaller);
+}
+
+.thumbnailNonImage::before {
+  content: "";
+  display: block;
+  position: absolute;
   right: 0;
   bottom: 0;
+  z-index: 1;
+  width: 20px;
+  height: 20px;
+  background: linear-gradient(
+    270deg,
+    rgb(244, 244, 244) 38.46%,
+    rgba(244, 244, 244, 0) 100%
+  );
+}
+
+.thumbnail .fileContentWrapper {
```

Comment:
If you move this `overflow` up to the FormatFile wrapper then you can use the previous structure for FormatFile--imageBlock for the image and the icons and won't need to absolute position another container that just clips the text.

Reply:
Adjusting this results in the below:
![Screen Shot 2022-05-27 at 11 33 00 AM](https://user-images.githubusercontent.com/104797704/170732331-34e4a692-ef90-4c6c-b3d1-ba14521a974f.png)